.scalafmt.conf

```diff
@@ -2,8 +2,9 @@ align = some
 danglingParentheses        = true
 indentOperator             = spray
 maxColumn                  = 120
-lineEndings                = unix
+lineEndings                = preserve
```

Comment:
This was needed to prevent the scalaformatter from changing my line endings from CRLF to LF on windows.